run_data.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+CLI Functionality to generate reports and summaries of data sources.
+
+To generate model output, see ./run_model.py
+"""
+import pathlib
+import json
+import logging
+import click
+from libs import build_params
+from libs.datasets import JHUDataset
+from libs.datasets.dataset_utils import AggregationLevel
+
+WEB_DEPLOY_PATH = pathlib.Path("../covid-projections/public/data")
+
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group()
+def main():
+    pass
+
+
+@main.command("latest")
+@click.option(
+    "--deploy",
+    is_flag=True,
+    help="Output data files to public data directory in local covid-projections.",
+)
+def run_latest(deploy=False):
+    """Get latest case values from JHU dataset."""
+    output_dir = pathlib.Path(build_params.OUTPUT_DIR)
+    if deploy:
+        output_dir = WEB_DEPLOY_PATH
+
+    output_folder = output_dir / "case_summary"
+    output_folder.mkdir(exist_ok=True)
+    timeseries = JHUDataset.local().timeseries().get_subset(None, country="USA")
+
+    timeseries.data.cases = timeseries.data.cases.fillna(0)
```

Comment:
if possible, it would be great that logic like formatting or transformation doens't live here

Reply:
Yeah definitely true.  

I've been punting on this issue for a bit - should we treat these missing values as zeros rather than NaNs across the board? If so, I can modify the timeseries class to fill those in as such